common/lcs/src/lib.rs

```diff
@@ -63,6 +64,24 @@
 //! |64-bit signed integer      |-1311768467750121216   |0xEDCBA98754321100 |[0011325487A9CBED] |
 //! |64-bit unsigned integer    |1311768467750121216    |0x12345678ABCDEF00 |[00EFCDAB78563412] |
 //!
+//! Internally, the LCS format also uses the concise [ULEB128 encoding](https://en.wikipedia.org/wiki/LEB128)
+//! for specific unsigned 32-bit integers, representing either a variable length or a tag in an enumeration.
```

Comment:
I think perhaps a better way of saying this is that LCS uses 32-bit integers to specify lengths for variable length data structures and indices (or variants) in an enumeration. Those values are stored using .....

Reply:
Let me improve the sentence a bit.

TBH I tried to get a version of the doc that was clear enough but not necessary final. I suspect we are going to iterate a lot on the docs afterwards anyway.